src/modules/history.js

```diff
@@ -0,0 +1,54 @@
+/**
+* history
+* @author Deux Huit Huit
+*/
+(function ($, undefined) {
+
+	'use strict';
+
+	var scope = $('#site');
+
+	var sels = {
+		input: '.js-history-input'
+	};
+
+	var urls = [];
+
+	var update = function () {
+		urls.push(window.location.href);
+		scope.find(sels.input).each(function () {
+			var t = $(this);
+			var value = urls.join('\n');
+
+			if (t.attr('data-history-unique') === 'true') {
+				value = _.uniq(urls).join('\n');
+			}
+
+			t.val(value);
+		});
+	};
+
+	var onPageEnter = function (key, data) {
+		update();
```

Comment:
Should we do it on leave ? Do we really want the current page to be in this ?

Reply:
It's technicaly a page the user saw so yeah I would keep it even if it will always be there.